src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,19 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        List<Integer> evenNumbers = numbers.stream()
```

Comment:
Try to get result from the stream
```suggestion
        return numbers.stream()
```